### PR TITLE
[master < T1156] Add Relationship.RemoveProperty in C++ API

### PR DESCRIPTION
**Pull request by imilinovic**
A need for RemoveProperty function was found when implementing custom C++ Query module. So it was implemented.
[master < Epic] PR
- [x] Check, and update documentation if necessary
- [X] Write E2E tests
- [ ] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch and the Epic branch
- [ ] Provide the full content or a guide for the final git message

[master < Task] PR
- [ ] Check, and update documentation if necessary
- [ ] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [X] Write a release note here, including added/changed clauses
Users can now use RemoveProperty() function for class Relationship when writing custom query modules in C++ API.
- [X] Tag someone from docs team in the comments

### CODE REVIEW

**Comment by imilinovic** [comment]
@vpavicic Will add to docs after it gets approved.

File: tests/unit/cpp_api.cpp [hunk 1]
```diff
@@ -574,6 +574,22 @@ TYPED_TEST(CppApiTestFixture, TestNodeRemoveProperty) {
   ASSERT_EQ(node.Properties().size(), 0);
 }
 
+TYPED_TEST(CppApiTestFixture, TestRelationshipRemoveProperty) {
+  mgp_graph raw_graph = this->CreateGraph(memgraph::storage::View::NEW);
+  auto graph = mgp::Graph(&raw_graph);
+  auto node_1 = graph.CreateNode();
+  auto node_2 = graph.CreateNode();
+  auto relationship = graph.CreateRelationship(node_1, node_2, "RELATIONSHIP");
```
Comment (antoniofilipovic) [inline tests/unit/cpp_api.cpp:582]: All good, but usually we have for relationship first letter in uppercase, `:Relationship`

File: include/mgp.hpp [hunk 2]
```diff
@@ -2645,6 +2648,8 @@ inline void Relationship::SetProperty(std::string property, Value value) {
   mgp::edge_set_property(ptr_, property.data(), value.ptr());
 }
 
+inline void Relationship::RemoveProperty(std::string property) { SetProperty(property, Value()); }
```
Comment (antoniofilipovic) [inline include/mgp.hpp:2651]: Is it possible to pass here only `{}` as an empty value

**Review by antoniofilipovic — APPROVED** [review]
Looks good, apply these two changes and it can be shipped

File: include/mgp.hpp (hunk 2 above)
Comment (imilinovic) [reply include/mgp.hpp:2651]: No:
`Chosen constructor is explicit in copy-initialization`

**Review by Josipmrden — APPROVED** [review]

**Comment by vpavicic** [comment]
@antoniofilipovic has this been merge to 2.10.1 or is this for 2.11?

**Pull request merged**